### ISSUE TITLE
Hierarchy within slugs should be reflected in related notes

### ISSUE THREAD

**Issue by cdata** [subconsciousnetwork/subconscious#993]
I have two notes. Their slugs are:

- `/cortex-prime`
- `/cortex-prime/mod/matching-energy`

Neither of the notes at these slugs appear in the "related notes" of the other. But, they are definitely related, and I would like for them to appear in each other's related notes.

**Comment by bmann** [comment]
I thought about doing this because I learned it from LogSeq. +1 for this.

**Comment by gordonbrander** [comment]
Agree. We included deep hierarchy in slashlinks in order to allow for this kind of faceting.
